view.py

```diff
@@ -0,0 +1,757 @@
+            self.window,
+            text="Restaurant Info",
+            font="none 12",
+            bg="#0080c0",
+            height=2,
+            width=15,
+            variable=self.var,
+            value=2,
+        )
+        radiobtn_menu = Radiobutton(
+            self.window,
+            text="Menus",
+            font="none 12",
+            bg="#0080c0",
+            height=2,
+            width=5,
+            variable=self.var,
+            value=3,
+        )
+        btn_list = Button(
+            self.window,
+            text="Show List",
+            font="none 12",
+            bg="light grey",
+            height=1,
+            width=10,
+            #command = a function in controller to return a list corresponding
+            #to the value of var, then output the information in the listbox
+        )
+        lbl_prompt = Label(bg="#0080c0", font="none 12", text="List: ")
+        self.view1_list_box = Listbox(
+            self.window, font="none 12", height=12, selectmode="SINGLE"
+        )
+        btn_info = Button(
+            self.window,
+            text="More info",
+            font="none 12",
+            bg="light grey",
+            height=1,
+            width=10,
+            command = self.controller.admin_view_more_info_btn
+        )
+        btn_update = Button(
+            self.window,
+            text="Update",
+            font="none 12",
+            bg="light grey",
+            height=1,
+            width=10,
+            #command = a function in controller to return a list of selected
+            #restaurant name, address, and its menu, then open a menu window
+            command = self.controller.request_menu
+        )
+        btn_exit = Button(
+            self.window,
+            text="Exit",
+            font="none 12",
+            bg="light grey",
+            height=1,
+            width=10,
+            command=self.controller.back_to_welcome
+        )
+        # ---DELETE ME---
+        self.view1_list_box.insert(1, "Press exit to loop")
+        # ---DELETE ME---
+        
+        Label(
+            bg="#0080c0", text="Administrative Function", font=self.font1
+        ).grid(row=0, columnspan=7)
+        Label(bg="#0080c0", height=2, width=5).grid(row=1, column=0)
+        radiobtn_id.grid(row=2, column=1)
+        radiobtn_info.grid(row=2, column=2)
+        radiobtn_menu.grid(row=2, column=3)
+        btn_list.grid(row=2, column=5, sticky="e")
+        Label(bg="#0080c0", height=2, width=5).grid(row=2, column=4)
+        lbl_prompt.grid(row=3, column=1, columnspan=3, sticky="w")
+        self.view1_list_box.grid(row=4, column=1, columnspan=4, sticky="we")
+        btn_info.grid(row=4, column=5, sticky="e")
+        Label(bg="#0080c0").grid(row=5, columnspan=7)
+        btn_update.grid(row=6, column=2)
+        btn_exit.grid(row=6, column=5)
+
+    def _rest_info_place_widget(self):
+        self.rest_info_list = self.controller.restaurant_info()

```

Comment:
rest_info_list should be an argument and not a call within the controller. 
I'm not sure if you need to define it as a class variable with self. That unless you intend to use it in other methods with the given stored data.